src/uu/stat/src/stat.rs

```diff
@@ -529,7 +543,6 @@ impl Stater {
         }
         Ok(tokens)
     }
-
```

Comment:
I don't understand why you remove this empty line, it separates the two functions.